### PR TITLE
Add clearer error message

### PR DESCRIPTION
**Pull request by ggetz**
Fixes ##6399 with a clearer error.

Opened https://github.com/AnalyticalGraphicsInc/cesium/issues/6404 for a better fix by using the legacy enabled build of the draco decoder module.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@ggetz, thanks for the pull request! Maintainers, we have a signed CLA from @ggetz, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/DracoLoader.js [hunk 1]
```diff
@@ -177,6 +177,12 @@ define([
             return when.resolve();
         }
 
+        if (FeatureDetection.isInternetExplorer()) {
+            return when.reject({
+                message : 'Draco decoding is not supported in legacy environments.'
```
Comment (hpinkos) [inline Source/Scene/DracoLoader.js]: To be a little more obvious, maybe just say "Draco decoding is not currently support in Internet Explorer"

**Comment by hpinkos** [comment]
@ggetz Add a comment about this in `CHANGES.md` too.  And link to the issue about adding support in the future.

**Comment by ggetz** [comment]
Thanks @hpinkos, updated!

**Comment by hpinkos** [comment]
One more thing (sorry for not thinking of this before)
Add an `isSupported` attribute, similarly to [GroundPrimitive.isSupported](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/GroundPrimitive.js#L365)

**Comment by hpinkos** [comment]
And probably also mention the `isSupported` attribute in CHANGES.md

**Comment by ggetz** [comment]
`DracoLoader` isn't part of the public API, should it still have the `isSupported` attribute? Should I add a `isDracoDecodingSupported` attribute to `Model`?

**Comment by hpinkos** [comment]
Oooh, gotcha.  Yeah, nevermind then, I think this is fine =)

**Comment by hpinkos** [comment]
Thanks @ggetz!

**Pull request merged**